tests/general/InvalidLinkBearTest.py

```diff
@@ -149,12 +149,16 @@ def test_link_ignore_regex(self):
         ignored_URLs = """
         http://sub.example.com
         http://sub.example.com/something
+        http://www.%s.com
+        http://www.%d.com
+        http://www.%f.com
         """.splitlines()
 
         not_ignored_URLs = """
         http://www.notexample.com
         http://exampe.com
         http://example.co.in
+        http://www.google.com/hello%20world
```

Comment:
Oh sorry I think these links should go up into `test_run`,  the `%s` links shall go below `# Not a link` and the google one into the invalid links. 
Sorry again :/